implementations/micrometer-registry-otlp/src/test/java/io/micrometer/registry/otlp/OtlpConfigTest.java

```diff
@@ -276,4 +277,25 @@ void histogramPreferenceUseEnvVarWhenConfigNotSet() throws Exception {
                 .isEqualTo(HistogramFlavor.BASE2_EXPONENTIAL_BUCKET_HISTOGRAM));
     }
 
+    @Test
+    void histogramFlavorPerMeter() {
+        Map<String, String> properties = new HashMap<>();
+        properties.put("otlp.histogramFlavorPerMeter",
+                "a.b.c=explicit_bucket_histogram ,expo =base2_exponential_bucket_histogram");
```

Comment:
Is spacing intentional?
```suggestion
                "a.b.c=explicit_bucket_histogram, expo=base2_exponential_bucket_histogram");
```

Reply:
Yes, this was intentionally testing that it can handle weird spacing. That could be made more explicit in a separate test perhaps.